### ISSUE TITLE
Install routine UI: improve text contrast

### ISSUE THREAD

**Issue by math-GH** [piwik/piwik#17785]
Install routine: text color

![grafik](https://user-images.githubusercontent.com/1645099/126037095-b39e5f34-1b9c-4585-bc8d-99295b1439e2.png)
(checked with the development tool in Firefox)

## Expected Behavior
darker grey (better: black), that has a much better contrast

## Current Behavior
grey font color with low contrast.
font-color: #858585 (contrast ratio: 3.7)

## Possible Solution
font-color: #757575 (contrast ratio: 4.5 => minimum ratio https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Perceivable/Color_contrast)

## Your Environment
* Matomo Version: 4.3.1
* Browser: Firefox 90
